File: backend/components/metiers/threed/security/docs/samples/onboarding/samples/sample_onboarding.py
# sample_onboarding.py
# Exemple d’intégration avancée pour l’onboarding

from helpers_onboarding import start_onboarding, complete_onboarding

user = {'id': 'u123', 'name': 'Alice'}
user = start_onboarding(user)
print('Started:', user)
user = complete_onboarding(user)
print('Completed:', user)

File: backend/components/metiers/threed/security/docs/samples/onboarding/samples/sample_onboarding.js
// sample_onboarding.js
// Exemple d’intégration avancée pour l’onboarding

const { startOnboarding, completeOnboarding } = require('./helpers_onboarding');

let user = { id: 'u123', name: 'Alice' };
user = startOnboarding(user);
console.log('Started:', user);
user = completeOnboarding(user);
console.log('Completed:', user);
